competencies/roles/DataScienceLevelMapping.md

```diff
@@ -0,0 +1,23 @@
+The three categories of competencies can be used to map to typical job titles specific roles. 
+
+This framework also uses the [Developer](./Developer.md)  competencies of 
+"Tech Design, Implementation and Code Ownership" and of "System Design" to create 
+a role family called Machine Learning Engineer
+
+
+| Title/Job                   | Key Competenies           | Other Competencies           |     Notes  |
+| --------------------------- |:-------------------------:| ----------------------------:| ----------:|
+| Data Science I (junior)     |                           | 90% emerging; 10% proficient |  New Grad, or first one or two years |
+| Data Science II             | Proficient: "Machine Learning Analysis"      |  50% emerging, 50% proficient  |   |
+| Data Science III a (senior) | Proficient: "Error, Causality, Inference" | 60%% proficient; some authority  |  | 
+| Data Science III b (senior) |                           | 90% proficient; some authority  | High end to end impact; Strength: "Business Alignment" and "Communication"  | 
+| Principle Data Scientist    | All DS specific | 50% Proficient; 50% Authority | Strong self awareness, Strong on general areas such as business acumen | 
+
+
+
+| Title/Job                   | Key Competenies           | Other Competencies           |     Notes  |
+| --------------------------- |:-------------------------:| ----------------------------:| ----------:|
+| Machine Learning Engineer III  | Proficient: "Machine Learning Analysis"  and "Tech Design, Implementation and Code Ownership"   |  50% emerging, 50% proficient  |  Comparable to SWDev III |
```

Comment:
I originally read this as a senior role, given "Data Science III" is a senior role (and realized it was more of an intermediate role after comparing the Developer mappings).  The naming could be confusing given the numbering we're using for Data Science vs Developer.  I don't really have a better suggestion though, aside from maybe using Data Science III / Data Science IV for consistency (rather than a / b), and speaking of roles as "Junior (1/2),  Intermediate, Senior (1/2)" for clarity